src/lib/themes.js

```diff
@@ -38,21 +47,21 @@ export const normalizeTheme = (data = {}) => {
   const theme = {
     colors: normalizeThemeColors(data.colors, defaultTheme.colors),
     images: {
-      additional_backgrounds: []
+      additional_backgrounds: [ "" ]
```

Comment:
Why an empty string instead of just an empty array?

Reply:
Wanted to ensure there's always a 0 element that's defined

Reply:
How does Firefox handle this